.github/scripts/toolchain/install-cygwin-headers.sh

```diff
@@ -5,7 +5,7 @@ source `dirname ${BASH_SOURCE[0]}`/../config.sh
 CYGWIN_SOURCE_PATH=$SOURCE_PATH/cygwin
 
 echo "::group::Install Cygwin headers"
-    mkdir -p $TOOLCHAIN_PATH/include
+    mkdir -p $TOOLCHAIN_PATH/$TARGET/include
```

Comment:
This is just oversight made previously. Does not really change anything and can be filed as a separate PR.